pandas/tests/extension/base/methods.py

```diff
@@ -31,3 +31,14 @@ def test_count(self, data_missing):
     def test_apply_simple_series(self, data):
         result = pd.Series(data).apply(id)
         assert isinstance(result, pd.Series)
+
+    @pytest.mark.parametrize('box', [pd.Series, lambda x: x])
+    @pytest.mark.parametrize('method', [lambda x: x.unique(), pd.unique])
+    def test_unique(self, data, box, method):
+        duplicated = box(type(data)([data[0], data[0]]))
```

Comment:
should this one also be `_constructor_from_sequence`? (as it is a generic test to be subclassed)